### ISSUE TITLE
Find and replace option

### ISSUE THREAD

**Issue by VigneshChennai** [chergert/gnome-builder#7]
Hi, I can't able to find the "find and replace" option in the builder. Is it not implemented yet ?

**Comment by chergert** [comment]
It's not implemented yet. We will probably bring the UI over from Gedit soon. If someone signs up to do that work, it will happen faster.

**Comment by fedelibre** [comment]
@chergert any update on this?
This is quite an important feature.

[Issue 754334](https://bugzilla.gnome.org/show_bug.cgi?id=754334#c2) on Gnome bugzilla is quite related. Why are you keeping two trackers for the same project? Isn't it confusing?

**Comment by chergert** [comment]
Yes it is confusing, but we don't have a way to redirect people from github to bugzilla. This repo isn't even up to date. git.gnome.org is the repository of record and bugzilla is the bugtracker of record.

I'm busy on cleaning up the plugin story and getting perspectives landed. I was really hoping this could be a task for someone joining the project since it is primarily just time consuming rather than difficult.

**Comment by coloradomountain** [comment]
this has not been resolved yet? I'll look into adding it.

**Comment by chergert** [comment]
There is a good chance we'll have someone implementing this as part of GSoC this summer (for Builder 3.22).

Plenty of stuff to work on though: https://wiki.gnome.org/Apps/Builder/Roadmap

**Comment by coloradomountain** [comment]
I already implemented it on my local version of gnome-builder. What did you mean by "It's not implemented yet. We will probably bring the UI over from Gedit soon. If someone signs up to do that work, it will happen faster."   do you mean your bringing over the features from gedit, or the full UI?

**Comment by chergert** [comment]
We want to explore new UI for this feature. I think mockups are in the works now.
